js/scenes/FTU.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Image,
+  Text,
+  Button,
+  Dimensions,
+  StyleSheet,
+  Platform,
+} from 'react-native';
+import ViewPager from 'react-native-viewpager';
+import ViewPagerIndicator from '../components/ViewPagerIndicator';
+import { connect } from 'react-redux';
+import { theme } from '../../config';
+
+export class FTU extends Component {
+  constructor(props) {
+    super(props);
+
+    this.navigator = this.props.navigator;
+
+    const PAGES = [
```

Comment:
Can we render the pages on demand? Or are we going to force to show always the two pages?